app/pages/common/ManualOrUploadPage.scala

```diff
@@ -48,6 +49,7 @@ case class ManualOrUploadPage(eventType: EventType, index: Int) extends Question
       case (Event23, _, Some(FileUpload)) => pages.common.FileUploadWhatYouWillNeedPage(eventType)
       case (Event23, index, Some(Manual)) => pages.event23.WhatYouWillNeedPage(index)
       case (Event24, index, Some(Manual)) => MembersDetailsPage(Event24, index)
+      case (Event24, _, Some(FileUpload)) => UploadingFileInfoPage(eventType)
```

Comment:
Can we use the common `FileUploadWhatYouWillNeedPage` for Event24, or are they sufficiently different?

Reply:
This is using the new design which splits out the `WhatYouWillNeed` page from the `UploadInfo` page. I assume we'll be doing this for all the events at some point but for now I think only Event 24 has this.